modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/QuerySpecTests.java

```diff
@@ -63,7 +70,58 @@ public static void afterClass() throws Exception {
         searchRequestParsers = null;
     }
 
-    // TODO add some sort of roundtrip testing like we have now for queries?
+    public static RatedRequest createTestItem(List<String> indices, List<String> types) {
+        String specId = randomAsciiOfLength(50);
+
+        SearchSourceBuilder testRequest = new SearchSourceBuilder();
+        testRequest.size(23);
```

Comment:
Any reason for this size here? I don't think we need to test this randomly (this is done already in the SearchSourceBuilder tests), but this special size caught my attention. Maybe just ommit and use the default or add a comment?